src/envoy/http/service_control/http_call.cc

```diff
@@ -101,8 +102,13 @@ class HttpCallImpl : public HttpCall,
 
         ENVOY_LOG(debug, "http call response status code: {}, body: {}",
                   status_code, body);
-        on_done_(Status(Code::INTERNAL, "Failed to call service control"),
-                 body);
+        std::string error_msg =
+            absl::StrCat("Calling ServiceControl failed with: ", status_code);
```

Comment:
Since this error message will be exposed to API consumers, let's try to reduce confusion and say "Google Service Control API" instead of "ServiceControl".

Reply:
Done